[2022-06-15 23:06:39,351] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: say_hi.get_teams_task scheduled__2022-06-13T00:00:00+00:00 [queued]>
[2022-06-15 23:06:39,366] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: say_hi.get_teams_task scheduled__2022-06-13T00:00:00+00:00 [queued]>
[2022-06-15 23:06:39,366] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-15 23:06:39,366] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-15 23:06:39,366] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-15 23:06:39,392] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): get_teams_task> on 2022-06-13 00:00:00+00:00
[2022-06-15 23:06:39,399] {standard_task_runner.py:52} INFO - Started process 183 to run task
[2022-06-15 23:06:39,410] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'say_hi', 'get_teams_task', 'scheduled__2022-06-13T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/username.py', '--cfg-path', '/tmp/tmplve_c8lm', '--error-file', '/tmp/tmp2rrf7eq0']
[2022-06-15 23:06:39,411] {standard_task_runner.py:80} INFO - Job 3: Subtask get_teams_task
[2022-06-15 23:06:39,608] {task_command.py:370} INFO - Running <TaskInstance: say_hi.get_teams_task scheduled__2022-06-13T00:00:00+00:00 [running]> on host 6a14fd44e8dc
[2022-06-15 23:06:39,767] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=say_hi
AIRFLOW_CTX_TASK_ID=get_teams_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-13T00:00:00+00:00
[2022-06-15 23:06:39,769] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-15 23:06:39,770] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "getting three random teams"']
[2022-06-15 23:06:39,790] {subprocess.py:85} INFO - Output:
[2022-06-15 23:06:39,805] {subprocess.py:92} INFO - getting three random teams
[2022-06-15 23:06:39,806] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-15 23:06:39,894] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=say_hi, task_id=get_teams_task, execution_date=20220613T000000, start_date=20220615T230639, end_date=20220615T230639
[2022-06-15 23:06:39,942] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-15 23:06:40,149] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
